src/lib/components/Input.tsx

```diff
@@ -144,8 +139,26 @@ const decimalEnforcer = (nextUserInput: string) => {
   }
   return null
 }
+
 export const DecimalInput = forwardRef(function DecimalInput(props: NumericInputProps, ref) {
-  return <NumericInput pattern="^[0-9]*[.,]?[0-9]*$" enforcer={decimalEnforcer} ref={ref as any} {...props} />
+  return (
+    <NumericInput
+      // universal input options
```

Comment:
These options should be added to the `NumericInput` internal component, not to the `DecimalInput` directly.

Reply:
This is done already on L103. I think you should add maxLength to that list, and otherwise revert this line.